### ISSUE TITLE
"LMZAError: Input format not supported by decoder" on startup

### ISSUE THREAD

**Issue by KiraGreifeneder** [vial-kb/vial-gui#163]
File "main_window.py", line 270, in on_devices_updated
  File "main_window.py", line 274, in on_device_selected
  File "autorefresh\autorefresh.py", line 79, in select_device
  File "vial_device.py", line 51, in open
  File "protocol\keyboard_comm.py", line 79, in reload
  File "protocol\keyboard_comm.py", line 135, in reload_layout
  File "lzma.py", line 334, in decompress

LZMAError: Input format not supported by decoder

**Comment by johnmcclain77** [comment]
Same error as well

**Comment by LukeDRussell** [comment]
I'm getting this on Fedora 40 with Gnome. I wasn't getting the error while on Plasma.

**Comment by baldur132** [comment]
Yes, this. It worked on my machine at home, but doesn't at on my work machine.

**Comment by oXCookiXo** [comment]
getting the same error with web or desktop

**Comment by mysonthebison** [comment]
Same error as well with a Kit Adam keyboard

**Comment by jasonmaskell** [comment]
I'm having this issue also, with Ergodash keyboard and Helios controller. It works fine at home where I installed all the software to flash and build the firmware. But then on my work keyboard, both through the browser and the portable app, I get this error. Home is Windows 11, work is Windows 10.

**Comment by rkristelijn** [comment]
I'm working on lily58, macbook m2 with Dell WD19T dock.

I can confirm:
- vial throws error if lily58 is connected via the dock
- directly on macbook it works with no problems

Would be great if this could be fixed <3

**Comment by leigholiver** [comment]
WD19TB dock here, in case it helps anyone else i can connect my keyboard to Vial if I plug into the thunderbolt port on the dock rather than a usb-c port.
